source/Calamari.Tests/Fixtures/Manifest/ExecuteManifestCommandFixture.cs

```diff
@@ -56,8 +56,7 @@ public void WithInstructions()
             {
                 var generatedApplicationPath = CodeGenerator.GenerateConsoleApplication("node", temporaryDirectory.DirectoryPath);
 
-                var destinationPath =
-                    CalamariEnvironment.IsRunningOnWindows ? Path.Combine(temporaryDirectory.DirectoryPath, "app") : Path.Combine(temporaryDirectory.DirectoryPath, "app", "bin");
+                var destinationPath = Path.Combine(temporaryDirectory.DirectoryPath, "app");
```

Comment:
@andrewabest  The paths are different in Linux vs Windows, did u test this in Linux?